empty-project/Assets/TestScript.cs

```diff
@@ -36,7 +36,8 @@ public class TestScript : MonoBehaviour {
 			Debug.Log ("Child removed!");
 		};
 
-		//firebase.SetValue ("working?");
+		firebase.SetValue ("SetValue working?");
```

Comment:
There was an issue on reading Serialized JSON as a string. You have to make
sure to use the Datasnapshot.DictionaryValue instead of String Value
otherwise it causes that ios issue you helped me fix. That's all. Works
like a charm for the rest.
On Thu, Mar 31, 2016 at 6:13 PM benwulfe <notifications@github.com> wrote:

> In empty-project/Assets/TestScript.cs
> <https://github.com/firebase/Firebase-Unity/pull/29#discussion_r58136662>:
>
> > @@ -36,7 +36,8 @@ public class TestScript : MonoBehaviour {
> >  			Debug.Log ("Child removed!");
> >  		};
> >
> > -		//firebase.SetValue ("working?");
> > +		firebase.SetValue ("SetValue working?");
>
> so there was an issue with setJSON -- was that fixed? Is this PR ready to
> go?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/firebase/Firebase-Unity/pull/29/files/b809144708513a7afd3f0a967a9bcd4e8bf3f603#r58136662>
>